### ISSUE TITLE
add support for vs2022

### ISSUE THREAD

**Issue by softboy99** [cnsharp/nupack#31]
add support for vs2022

**Comment by Krisser50** [comment]
Yes please! I use NuPack a lot, and would be happy to use it in VS 2022 👍

**Comment by jeremychapman** [comment]
This is the best VS  extension for deploying packages in my opinion.  Simple and straightforward.  Needs VS 2022 support

**Comment by KunleSmart** [comment]
Please, add support for VS2022. Thank you in advance

**Comment by qcnguyen** [comment]
Yes please add support for vs 2022

**Comment by ldqk** [comment]
Yes please add support for vs 2022

**Comment by czh0221** [comment]
Yes please! I use NuPack a lot, and would be happy to use it in VS 2022 👍, I like it.  simple but great👍.
.

**Comment by xunqinbo** [comment]
Thanks for your affirmation.I left the.NET world for a few years for a few reasons.And now I am back.Please try the latest preview version.
[NuPack 2022](https://marketplace.visualstudio.com/items?itemName=CnSharpStudio.nupack2022).
